src/app/lib/logger.const.js

```diff
@@ -17,6 +17,10 @@ const logCodes = {
   MEDIA_ASSET_REVOKED: 'media_asset_revoked',
   MEDIA_ASSET_EXPIRED: 'media_asset_expired',
   MEDIA_METADATA_UNAVAILABLE: 'media_metadata_unavailable',
+  CPS_MEDIA_WITHOUT_IMAGE: 'cps_media_without_image',
+  CPS_MEDIA_WITHOUT_ID: 'cps_media_without_id',
+  CPS_MEDIA_WITHOUT_FORMAT: 'cps_media_without_format',
+  CPS_MEDIA_WITHOUT_VERSION_ID: 'cps_media_without_version_id',
```

Comment:
I think it'd be better to have a single `CPS_MEDIA_MISSING_FIELD` event and pass in the missing field when it is called.
```suggestion
  CPS_MEDIA_MISSING_FIELD: 'cps_media_missing_field',
```
This will help when viewing in our logs - we can have a single view for missing fields and then have the field defined as a separate value.